### ISSUE TITLE
Wishlist: swap controller hand assignments

### ISSUE THREAD

**Issue by wstephenson** [LibreVR/Revive#390]
It's hard to know which Vive wand will be seen as the left Oculus Touch controller and vice versa without picking everything up.  Since the Touch controllers are hand specific, swapping them in software (eg Medium) does not give the desired results.  It would be nice to have a gesture like in Tilt brush, tapping the bases of the wands together, that swaps how the wands are presented to the Oculus runtime.

**Comment by LibreVR** [comment]
Revive already uses the OpenVR left/right assignment same as every other Vive game. I don't see why this a problem Revive needs to solve. It would be better if this were implemented in SteamVR.

**Comment by LibreVR** [comment]
Also note that SteamVR is able to detect assign these roles pretty well by itself. Just make sure you pick up the controllers before starting a game.

Or if that's not possible, only turn the controllers on after you've picked everything up.
